src/Core/Prefab/GlobeView.js

```diff
@@ -18,6 +18,74 @@ import { updateLayeredMaterialNodeImagery, updateLayeredMaterialNodeElevation }
 import { globeCulling, preGlobeUpdate, globeSubdivisionControl, globeSchemeTileWMTS, globeSchemeTile1 } from '../../Process/GlobeTileProcessing';
 import BuilderEllipsoidTile from './Globe/BuilderEllipsoidTile';
 
+/**
+ * Emit when the globe is initialized
```

Comment:
What does that exactly means? (what is initialized? What can the user expect when this fires?)